judo-runtime-core-bootstrap-postgresql/src/main/java/hu/blackbelt/judo/runtime/core/bootstrap/dao/rdbms/postgresql/JudoPostgresqlModules.java

```diff
@@ -55,18 +65,25 @@ public static class JudoPostgresqlModulesBuilder {
     }
 
     @Builder
-    private JudoPostgresqlModules(String host, Integer port, String user, String password, String databaseName, Integer poolSize) {
+    private JudoPostgresqlModules(String host, Integer port, String user, String password, String databaseName, Integer poolSize, Dialect dialect, DataSource source, PostgresqlRdbmsInit postgresqlRdbmsInit) {
         this.host = host;
         this.port = port;
         this.user = user;
         this.password = password;
         this.databaseName = databaseName;
         this.poolSize = poolSize;
+        this.dialect = dialect;
+        this.source = source;
+        this.postgresqlRdbmsInit = postgresqlRdbmsInit;
     }
 
     @Override
     protected void configureDialect() {
-        bind(Dialect.class).toInstance(new PostgresqlDialect());
+        if(dialect != null) {
+            bind(Dialect.class).toInstance(dialect);
+        }else {
+            bind(Dialect.class).toInstance(new PostgresqlDialect());
+        }
     }
```

Comment:
In postgreSql the dialect HAVE TO BE PostresqlDialect.

Reply:
[06d0feb](https://github.com/BlackBeltTechnology/judo-runtime-core/pull/148/commits/06d0febfc2bca7ca64bda4d16aec4a828197c7c3)